html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <div>
+        <label for="email">Email:</label> <br>
+        <input type="text" name="email">
+      </div>
+      <div>
+        <label for="cool">Why is coding cool?</label> <br>
+        <select name="cool">
+          <option value="option">Select an option</option>
+          <option value="interest">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="power">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="language">Favorite language (choose one)</label> <br>
+        <input type="radio" name="language" value="html"> HTML
+        <input type="radio" name="language" value="css"> CSS
+        <input type="radio" name="language" value="javascript"> JavaScript
+      </div>
+      <div>
+        <label for="info">Would you  like to sign up for more information?
+        </label> <br>
+        <input type="checkbox" name="info" value="signup"> Sign me up!
+      </div>
+      <div>
+        <label for="comments">Comments</label> <br>
```

Comment:
This `for` attribute here is correct, but the form control element it is paired with needs a matching `id` attribute with the same value.